.github/workflows/check+deploy.yaml

```diff
@@ -20,6 +20,18 @@ jobs:
                 submodules: true
                 path: _src/
 
+            - name: Verify release version matches source code version
+              if: github.event == 'release' && startsWith(github.ref, 'refs/tags/v')
+              shell: bash
+              run: |
+                pushd _src
+                export TAG_VERSION=${GITHUB_REF##refs/tags/v}
+                export SRC_VERSION=$(sed -ne "s/^__version__\s*=\s*'\([^']*\)'/\1/p" wilson/_version.py)
+                if [[ ${TAG_VERSION} != ${SRC_VERSION} ]] ; then
+                  exit 1
+                fi
+                popd
```

Comment:
(The different steps all run from the working directory, i.e. `cd/pushd` won't affect others. Also I don't think you need the `export`)

Reply:
Also the other `pushd`/`popd` could be removed as well

Reply:
I developed a habit to use the ``pushd``/``popd`` in case I have to change to a different directory. A ``cd`` would be needed anyway, since the source is checkout out into ``INITIAL_WORKING_DIRECTORY/_src``.

Reply:
The most common thing seems to be to set the `working-directory`  of the gh action step, but I'm just bikeshedding here :)